test/JDBC/expected/BABEL-5119-vu-verify.out

```diff
@@ -0,0 +1,186 @@
+-- tsql
+CREATE LOGIN login_babel5119_1 WITH PASSWORD = '12345678'
+CREATE LOGIN login_babel5119_2 WITH PASSWORD = '12345678'
+ALTER authorization on database::BABEL5119_1 TO login_babel5119_1
+GO
+-- terminate-tsql-conn
+
+-- tsql user=login_babel5119_1 password=12345678
+USE master
+GO
+CREATE TABLE BABEL5119_t4(a int)
+GO
+CREATE TABLE BABEL5119_t5(a int)
+GO
+CREATE TABLE BABEL5119_t6(a int)
+GO
+INSERT INTO BABEL5119_t4 VALUES(1)
+GO
+~~ROW COUNT: 1~~
+
+CREATE VIEW BABEL5119_v4 AS SELECT 1
+GO
+CREATE VIEW BABEL5119_v5 AS SELECT 1
+GO
+CREATE VIEW BABEL5119_v6 AS SELECT 1
+GO
+CREATE PROCEDURE BABEL5119_p4 AS SELECT 1
+GO
+CREATE PROCEDURE BABEL5119_p5 AS SELECT 1
+GO
+CREATE PROCEDURE BABEL5119_p6 AS SELECT 1
+GO
+
+-- Checking crossdb for sys views
+USE BABEL5119_1
+GO
+
+SELECT name FROM master.sys.databases WHERE name LIKE 'BABEL5119%'
+ORDER BY name
+GO
+~~START~~
+varchar
+babel5119_1
+~~END~~
+
+
+SELECT name FROM master.sys.views WHERE name LIKE 'BABEL5119%'
```

Comment:
What are we trying to check with this views output?
Please write one line comment for each cases that what are we trying to verify here.